components/atom/input/src/index.scss

```diff
@@ -0,0 +1,7 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import "~@s-ui/react-molecule-field/lib/index.scss";
+@import "~@s-ui/react-form-input/lib/index.scss";
+
+.sui-AtomInput {
```

Comment:
Remove this unnecessary code. @imports will be executed on build and not used.
(don't remove the file)